modules/data_team_helper/php/NDB_Form_data_team_helper.class.inc

```diff
@@ -261,9 +284,16 @@ class NDB_Form_Data_Team_Helper extends NDB_Form
             $where        .= " AND s.Visit_label = :VL";
             $qparams['VL'] = $visit_label;
         }
-        if (($site !=null)&&($site!='All Sites')) {
-            $where         .= " AND s.CenterId = :CID";
-            $qparams['CID'] = $site;
+        if ($site!='All Sites') {
+            if ($site == 'All User Sites') {
+                $user           = User::singleton();
+                $siteID         = $user->getCenterIDs();
+                $where         .= " AND FIND_IN_SET(psc.CenterID, :CID)";
+                $qparams['CID'] = implode(',', $siteID);
+            } else {
```

Comment:
@MounaSafiHarab I think that else is required when we select a particulars site from the drop-down. If All sites -> all data 
If User Site-> all data of user sites(" AND FIND_IN_SET(psc.CenterID, :CID)";)
Else-> the data of the site selected.(" AND s.CenterID = :CID";)